### ISSUE TITLE
Data Plane RBAC with DisableLocalAuth- roles not honored

### ISSUE THREAD

**Issue by MoaidHathot** [Azure/cosmos-explorer#924]
I'm using CosmosDB Data Plane RBAC and wanted to enable `disableLocalAuth`, but once I enable `disableLocalAuth` I can't query Cosmos at all, even when I have sufficient role definitions. 
Every time I try to query, I receive the following error `Local Authorization is disabled. Use an AAD token to authorize all requests`.

From Issue [#853](https://github.com/Azure/cosmos-explorer/issues/853) and the following [comment ](https://github.com/Azure/cosmos-explorer/issues/853#issuecomment-852392265)

> That's right, in Cosmos Explorer we try to run listKeys, if the logged in tenant is missing permissions to fetch the keys we'll fallback to AAD RBAC

I think the issue is due to `listKeys`. Is there a way to mitigate it? A solution could be to skip `listKeys` if `disableLocalAuth` is enabled.
What do you think?

**Comment by southpolesteve** [comment]
@MoaidHathot Are you still seeing this issue? We pushed some changes that should make this work correctly on cosmos.azure.com

**Comment by MoaidHathot** [comment]
Hi @southpolesteve thank you for the update and for the quick fix.
It seems the issue with `listKeys` has been fixed.  We are planning to do more tests on Sunday to verify different scenarios with RBAC and `disableLocalAuth`. I will update once we do :)

I really appreciate your help!

**Comment by MoaidHathot** [comment]
Hi @southpolesteve I'm still unable to create or query entries when `disableLocalAuth=true`. 
The `listKeys` error isn't showing anymore, but when I try to perform an action, I receive the following error message as popup `Unexpected end of JSON input`, and in the Network tab, I receive 401 from `https://main.documentdb.ext.azure.com/api/guest/runtimeproxy/authorizationTokens`.

Steps to reproduce
* Use Bicep to create a new CosmosDB resource with `disableLocalAuth=true`.
* Use Azure CLI/PowerShell Azure to assign role definition id `00000000-0000-0000-0000-000000000002` (Cosmos DB Built-in Data Contributor) to my User's ObjectId for the CosmosDB resource.
* Verify that the role assignment from the previous step was added successfully using Azure CLI/PowerShell Azure
* Go to `cosmos.azure.com`, log-in via Aad, select the newly created CosmosDB resource and try to interact with it (create new item or run a query)

I also tried to logout and login again from `cosmos.azure.com` but it didn't help.

Thank you for your help, I really appreciate the effort!
